test/cli.ts

```diff
@@ -1,76 +1,76 @@
-const tape = require('tape');
-const spawn = require('tape-spawn');
-const path = require('path');
-const pkg = require('../package.json');
+import * as tape from 'tape';
+import * as spawn from 'tape-spawn';
+import * as path from 'path';
+import * as pkg from '../package.json';
 
 tape('CLI', function (t) {
   t.test('--version', function (st) {
-    const spt = spawn(st, './solcjs --version');
+    var spt = spawn(st, './solc.js --version');
```

Comment:
Why did the name change here? That seems to be not related to typescript refactoring.

Reply:
Its not directly related with the typescript migration but having a clearly defined extension is important in my opinion.

```
"bin": {
    "solcjs": "dist/solc.js"
  },
```

The bin binding is the only important factor and this still remains solcjs, only changes internallly.

Reply:
Solved by #587.